### DISCUSSION TITLE
Here is my Public SSH key to add me into the repository

### DISCUSSION THREAD

**Discussion by JohnLSison** [srothetafesa/Tafe24S1T109_Ext#3] [General]
Hey Sam, here is my public key: ssh-rsa ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCkPdzLfQzgZdwXlvTwpeeRlyP+thaXv3PaYycgd7yeIUwfUP2IeCH+UWVgHNrDJr3wFnGJxhpjvC7hsfMBhOdd/x9qlR99WALYKDFu8XeI2s6hwEYT1053Lw5LI4b4vHJlrh+GAQeFX8TtI8KNI5lpeVJyM4bGSJsDvwuoKFEHNHA6DjAmXmFoezpnRPdWvmXaPL/mihmo1WAyMYL7O/7SxTpUxyzC7HvjX/iH54JiKyYzicAbwZfjakAjqF9/oVRIe3Koqjbud1Tfw++IanzRWXzApRCjGZXK90V/I977Ri1dXzoWXLL/Q+KBUOQEGda7HerHQE01VaoIdP4cGs33 rsa-key-20240319

**Comment by srothetafesa** [comment]
Thanks John, I will add your key to the repository.
